src/operator/concat.cc

```diff
@@ -46,29 +46,39 @@ Operator* ConcatProp::CreateOperatorEx(Context ctx, std::vector<TShape> *in_shap
 DMLC_REGISTER_PARAMETER(ConcatParam);
 
 MXNET_REGISTER_OP_PROPERTY(Concat, ConcatProp)
-.describe(R"code(Concate a list of array along a given axis.
+.describe(R"code(Join input arrays along the given dimension.
 
-The dimension sizes of the input arrays on the given axis should be the same.
+.. note:: `Concat` is deprecated use `concat`.
 
-For example::
+The dimension of the input arrays should be the same except along the dimension
```

Comment:
Fix to:

The shapes of the input arrays should be identical except for the dimension along which they will be concatenated.